script/test.sh

```diff
@@ -7,6 +7,5 @@ bundle exec jekyll build
 # Check for broken links and missing alt tags, ignore edit links to GitHub as they might not exist yet
 
 bundle exec htmlproofer \
-    --url-ignore "/github.com/(.*)/edit/" \
-    --url-ignore "/#/" \
```

Comment:
@ericherman We added this in #130 to unblock the new functionality added in #110, which is needed for #111 to render. @angelplasma says it can probably be made more precise - do think that's necessary?

Reply:
I removed it because it was not required to make any current link pass, and matching any link with an anchor tag or '#' in it is far too broad. Looking at other branches I saw that twitter links were failing, thus added that in to be ahead of that approaching need.

Indeed, if we need to introduce something to make links pass in another branch, let's see what sorts of links are failing and match something more specific than '#'.